### PR TITLE
re installed create-react-app

### PR DESCRIPTION
**Pull request by KenMan79**

### CODE REVIEW

**Review by andykeh710 — DISMISSED** [review]
looks good to me!

**Comment by skellet0r** [comment]
Nice @KenMan79. Question, is this related to a particular ticket? I'm confused to what this is in relation to. Could you link an issue, or describe what you've changed and why?

**Review by skellet0r — CHANGES_REQUESTED** [review]
It looks like you committed by accident the `.idea` directory which is a project configuration directory created by your IDE. You should add the .idea directory to the root `.gitignore` so as to prevent it's history being tracked in the repository.

File: west-texas-oil/src/App.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ function App() {
           target="_blank"
           rel="noopener noreferrer"
         >
-          Learn React
+          West Texas Oil
```
Comment (skellet0r) [inline west-texas-oil/src/App.js:18]: Looks like you only changed this one line @KenMan79 ?
I'll have to close this PR, when you have a more substantial change set submit a PR.
Or you can submit an Bug report to explain why only this one line change was necessary, and then re-submit the PR.

**Review by skellet0r — CHANGES_REQUESTED** [review]
Needs more, missing a bunch of meat on the bones.
Look to the feature tickets for tasks, or create a new feature request / bug report, and then work on that.

**Comment by KenMan79** [comment]
@skellet0r yes this is the wrong repository, this is for the other project related to tokens, I tried to find it so I could delete it but yes, thank you, and glad you like the start. It is written in React Native

**Pull request closed without merging**